index.bs

```diff
@@ -5365,7 +5364,7 @@ throws>CreateAlgorithmFromUnderlyingMethod ( <var ignore>underlyingObject</var>,
 <var>args</var> )</h4>
 
 <div class="note">
-  PromiseCall is a variant of <a>promise-calling</a> that works on methods.
+  PromiseCall is a variant of <a lt="call a user object's operation">promise-calling</a> that works on methods.
```

Comment:
> > Figure out what to do with "promise-calling".
> 
> I commented inline on some possible wording here.

@domenic What was your suggestion for promise-calling? I can't seem to find your comment. 😕

Reply:
Oops, it was in an unsubmitted review.

> Maybe something like "This encapsulates the relevant promise-related parts of the Web IDL [call a user object's operation] algorithm for use while we work on [moving to Web IDL](https://github.com/whatwg/streams/issues/963)."